packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item.tsx

```diff
@@ -66,17 +66,19 @@ export class SegmentedControlItem {
     validateProps(this, propTypes);
     // this additional validation is still needed because undefined is allowed with current propTypes
     throwIfPropIsUndefined(this.host, 'value', this.value);
+    const hasIcon = !!this.icon || !!this.iconSource;
```

Comment:
Boolean conversion can probably be skipped.
```suggestion
    const hasIcon = this.icon || this.iconSource;
```

Reply:
@denyo If we remove the boolean conversion here, then the value of `hasIcon` will be a string in a case `icon` or `iconSource` is defined. But the name `hasIcon` implicitly means "Boolean" and we use it as Boolean, so it's better to have explicit typing.

Besides of that, `getComponentCss` expects `hasIcon` to be Boolean and throws error `Argument of type 'string' is not assignable to parameter of type 'boolean'.` if I remove Boolean conversion.

Regarding  hasIcon in L86 - true, I will fix it

Reply:
Fixed hasIcon in L86